src/inet/network_interface/wifi_softap_espressif.c

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file network_interface_wifi_espressif.c
+ *
+ * @section License
+ * Copyright (C) 2016, Erik Moqvist
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERSOCKTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * This file is part of the Simba project.
+ */
+
+#include "simba.h"
+
+#if defined(ARCH_ESP)
+
+#include "esp_misc.h"
+#include "esp_wifi.h"
+#include "esp_softap.h"
+
+static struct softap_config softap_config;
+static struct ip_info ip_config;
+
+static err_t init()
+{
+    return (ERR_OK);
+}
+
+static int start(struct network_interface_t *netif_p)
+{
+    struct network_interface_wifi_softap_espressif_t *self_p;
+
+    self_p = container_of(netif_p,
+                          struct network_interface_wifi_softap_espressif_t,
+                          network_interface);
+
+    return (network_interface_wifi_softap_espressif_start(self_p));
+}
+
+static int stop(struct network_interface_t *netif_p)
```

Comment:
Is there any espressif function to stop the interface and disable the DHCP server?